roles/roles.py

```diff
@@ -0,0 +1,51 @@
+"""
+Definition of roles and their permissions
+"""
+
+from rolepermissions.roles import AbstractUserRole
+
+from roles.models import Role
+
+
+class Permissions:
+    """
+    All MicroMasters custom permissions
+    """
+    CAN_ADVANCE_SEARCH = 'can_advance_search'
+    CAN_EDIT_CMS = 'can_edit_cms'
+    CAN_MESSAGE_LEARNERS = 'can_message_learners'
+    CAN_CREATE_FORUMS = 'can_create_forums'
```

Comment:
the django-role-permissions docs specify that object permission checkers should go in `permissions.py`, but from what i've read, they don't say (a) that nothing else should go in there, or (b) that an internal vocabulary of permissions (which we have here) should go into `roles.py`. it's up to us, and to me it makes more sense for that vocabulary of permissions to be defined in `permissions.py`